### PR TITLE
mark vendor build targets manual

### PR DESCRIPTION
**Pull request by BenTheElder**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**: auto-tags the bazel `go_binary` and `go_test` targets in vendor/... as manual, so they won't be matched by `//...` unless they are a dependency of another target.

Additionally, fixes an odd file group in `staging/src/k8s.io/csi-api/pkg/crd/BUILD` that was throwing analysis errors.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**: this vendors a go tool "[buildozer](https://github.com/bazelbuild/buildtools/tree/master/buildozer)" from bazel in order to perform the auto-tagging.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/assign @ixdy 
cc @smarterclayton

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.11",
+	"GoVersion": "go1.12",
```
Comment (ixdy) [inline Godeps/Godeps.json]: :thinking:

File: hack/update-bazel.sh [hunk 2]
```diff
@@ -43,3 +44,11 @@ gazelle fix \
     "${KUBE_ROOT}"
 
 kazel
+
+# make targets in vendor manual
+# buildozer exits 3 when no changes are made ¯\_(ツ)_/¯
+# https://github.com/bazelbuild/buildtools/tree/master/buildozer#error-code
+buildozer 'add tags manual' '//vendor/...:%go_binary' '//vendor/...:%go_test' && ret=$? || ret=$?
```
Comment (ixdy) [inline hack/update-bazel.sh]: maybe also add `-quiet`?

File: Godeps/Godeps.json (hunk 1 above)
Comment (BenTheElder) [reply Godeps/Godeps.json]: removed

File: hack/update-bazel.sh (hunk 2 above)
Comment (BenTheElder) [reply hack/update-bazel.sh]: done

**Pull request merged**